.circleci/config.yml

```diff
@@ -834,6 +834,12 @@ commands:
             set -x
             docker push "$image"
             set +x
+            # Tag AC Version for non-dev images
+            if [[ "<< parameters.image_name >>" != *"dev"* ]]; then
+                ac_version=$(docker image inspect $image -f '{{index .Config.Labels "io.astronomer.docker.ac.version"}}')
```

Comment:
This will just be `1.10.7-7`, but we also need to include the distribtion in here too.

Reply:
fixed in https://github.com/astronomer/ap-airflow/pull/58/commits/9a2e658b594ad7ec58778f36c99b1925f708b8fb